artifacts/release/README.md

```diff
@@ -42,37 +34,23 @@ Note that `-B` flag will make assumptions about release version. Use it if your
 a special version, for example `8.0` you will want to remove `-B` and answer questions about versions in interactive
 mode that follows.
 
-## Compile binaries on multiple platforms
-
-Compile using maven on Windows, Linux and FreeBSD and upload to GH release
-
-```bash
-git fetch --tags
-git checkout tags/7.1.1
-mvn clean package -DskipTests -P build-web-console,build-binaries
-```
-
 ## Release Docker image
 
-Prune docker images to ensure clean build
+Azure [Build Docker Image](https://dev.azure.com/questdb/questdb/_build?definitionId=22) pipeline will automatically
+build and release the docker image to Docker Hub once the tag is pushed to the repo.
+See previous versions of this readme to find out how to manually build and push the image.
```

Comment:
do you mind keeping the manual commands in this version of the readme to save the operator having to use `git` in an event of CI failure

Reply:
Done in [a7020c7](https://github.com/questdb/questdb/pull/4931/commits/a7020c72bf729ca69510624bf9f1eb5bf9e5f81f)